pgmpy/models/BayesianNetwork.py

```diff
@@ -279,18 +279,35 @@ def get_cpds(self, node=None):
 
         Returns
         -------
-        A list of TabularCPDs: list
+        cpd : TabularCPD object or list of TabularCPD objects
+            If 'node' is specified, returns the 'TabularCPD' object corresponding to the node.
+            If 'node' is not specified, returns a list of all 'TabularCPD' objects added to the model.
+
+        Raises
+        ------
+        ValueError
+            If the specified node is not present in the model.
+
 
         Examples
         --------
         >>> from pgmpy.models import BayesianNetwork
-        >>> from pgmpy.factors.discrete import TabularCPD
-        >>> student = BayesianNetwork([('diff', 'grade'), ('intel', 'grade')])
-        >>> cpd = TabularCPD('grade', 2, [[0.1, 0.9, 0.2, 0.7],
-        ...                               [0.9, 0.1, 0.8, 0.3]],
-        ...                  ['intel', 'diff'], [2, 2])
-        >>> student.add_cpds(cpd)
-        >>> student.get_cpds()
+        >>> import pprint
+        >>> random_model = BayesianNetwork.get_random(n_nodes=8, edge_prob=0.4, n_states=2, latents=True)
```

Comment:
@B3CODER Here, you are currently using the `get_random` method to generate the model. I was suggesting to use the `get_example_model` method to generate the model. Something like this:
```python
from pgmpy.utils import get_example_model
model = get_example_model('cancer')
```
And then show examples similar to the ones that you have shown here.